draft-ietf-teep-protocol.md

```diff
@@ -642,6 +642,7 @@ update = [
     ? manifest-list => [ + bstr .cbor SUIT_Envelope ],
     ? attestation-payload-format => text,
     ? attestation-payload => bstr,
+    ? err-code => uint .size 1,
```

Comment:
You need a field description added around line 700 (after the attestation-payload field description)

Reply:
Thanks, will do.

Reply:
@dthaler I have added the description of err-code and err-msg.